File: test/cs/test_NetRadiationEquivalentEvaporation.cs
'Test generation'

public static class Test
{
    //first);
    //test1

    public static void test1()
    {
        NetRadiationEquivalentEvaporation res0 = Estimation_NetRadiationEquivalentEvaporation.CalculateNetRadiationEquivalentEvaporation(lambdaV:2.454,netRadiation:1.566);

        Console.WriteLine("{0}","netRadiationEquivalentEvaporation: "+res0.netRadiationEquivalentEvaporation);

        Console.WriteLine("netRadiationEquivalentEvaporation Comparison: ("+Math.Round(638.142,3)+";"+Math.Round(res0.netRadiationEquivalentEvaporation,3)+") "+Equals(Math.Round(638.142,3),Math.Round(res0.netRadiationEquivalentEvaporation,3)));
    }
}

File: test/java/test_NetRadiationEquivalentEvaporation.java
//Test generation'

import java.math.BigDecimal;
public static class Test
{
    //first);
    //test1);

    public static void test1()
    {
        double lambdaV = 2.454;
        double netRadiation = 1.566;

        NetRadiationEquivalentEvaporation res0 = Estimation_NetRadiationEquivalentEvaporation.CalculateNetRadiationEquivalentEvaporation(lambdaV,netRadiation);
        System.out.println(" netRadiationEquivalentEvaporation: "+res0.netRadiationEquivalentEvaporation);

        System.out.println(((new BigDecimal(res0.netRadiationEquivalentEvaporation)).setScale(3, BigDecimal.ROUND_HALF_DOWN)).equals((new BigDecimal(638.142)).setScale(3, BigDecimal.ROUND_HALF_DOWN)));
    }
}
